versioned_docs/version-5.x/macros-attributes/chain-extension.md

```diff
@@ -49,26 +49,39 @@ ink! smart contracts using this chain extension simply depend on this crate
 and use its associated environment definition in order to make use of
 the methods provided by the chain extension.
 
-## Attributes
+## Macro Attributes
+
+The macro supports only one required argument:
+ - `extension = N: u16`:
+The runtime may have several chain extensions at the same time. The `extension`
+identifier points to the corresponding chain extension in the runtime.
+The value should be the same as during the definition of the chain extension.
+You can consult the 
+[chain extension module documentation](https://paritytech.github.io/polkadot-sdk/master/pallet_contracts/chain_extension/index.html)
+if you are unsure how to find the chain extension code.
+Otherwise, you should contact the target chain's documentation 
+for specifications of any chain extensions it exposes.
+
+## Method Attributes
 
 There are two different attributes with which the chain extension methods
 can be flagged:
 
 | Attribute | Required | Default Value | Description |
 |:----------|:--------:|:--------------|:-----------:|
-| `ink(extension = N: u32)` | Yes | - | Determines the unique function ID of the chain extension method. |
+| `ink(function = N: u16)` | Yes | - | Determines the unique function ID within the chain extension. |
 | `ink(handle_status = flag: bool)` | Optional | `true` | Assumes that the returned status code of the chain extension method always indicates success and therefore always loads and decodes the output buffer of the call. |
 
 As with all ink! attributes multiple of them can either appear in a contiguous list:
 
 ```rust
 type Access = i32;
 
-#[ink::chain_extension]
+#[ink::chain_extension(extension = 12)]
```

Comment:
Do we have to consider this for the migration guide? Can you please take a look at the migration guide for chain extensions and see if there's anything to add/change?

Reply:
Added the notice in migration guide